docs/fields/file.md

```diff
@@ -32,5 +32,5 @@ field.
         attachment:
             type: file
             extensions: [ png, pdf ]
-            upload: portfolio
+        upload: portfolio
 ```
```

Comment:
Thanks, I thought I corrected that, but apparently I misunderstood the GitHub web interface.